core/che-core-api-core/src/main/java/org/eclipse/che/api/core/websocket/impl/BasicWebSocketEndpoint.java

```diff
@@ -106,7 +106,7 @@ public void onMessage(String message, Session session) {
     } else {
       combinedEndpointId = getOrGenerateCombinedEndpointId(session);
 
-      LOG.warn("Processing messing within unidentified session");
+      LOG.warn("Process interferes with an unidentified session");
```

Comment:
In terms of language the change is good, but the message itself is still not clear at all to me.